packages/carbon-graphs/tests/unit/controls/Scatter/ScatterLoad-spec.js

```diff
@@ -296,6 +296,100 @@ describe('Scatter - Load', () => {
         input.key,
       );
     });
+    it('does not update x axis range if allow calibration is disabled', () => {
+      const disableCalibrationInput = getAxes(axisDefault);
+      disableCalibrationInput.axis.x.allowCalibration = false;
+      disableCalibrationInput.axis.x.lowerLimit = 50;
```

Comment:
Is there a reason why only `lowerLimit` is being set here?

Reply:
No reason, it has been removed 25980a5